src/Factory/WorkerAbstractFactory.php

```diff
@@ -5,12 +5,20 @@
 use Interop\Container\ContainerInterface;
 use Laminas\EventManager\EventManager;
 use Laminas\EventManager\EventManagerInterface;
-use Laminas\ServiceManager\Factory\FactoryInterface;
+use Laminas\ServiceManager\Factory\AbstractFactoryInterface;
 use SlmQueue\Strategy\StrategyPluginManager;
 use SlmQueue\Worker\WorkerInterface;
 
-class WorkerFactory implements FactoryInterface
+/**
+ * TODO extend with deprecated WorkerFactory for compatibility
```

Comment:
Not sure if this is something you need to tackle now. Anyhow; just mentioning it, so you are aware of this!